service/src/test/java/bio/terra/cbas/dependencies/sam/TestSamService.java

```diff
@@ -176,13 +200,53 @@ void testGetSamUserExpiredToken() {
 
   @Test
   void testGetSamUserInterruptingToken() {
-    setTokenValue(tokenCausingInterrupt);
+    setTokenValue(tokenCausingUserInterrupt);
     SamInterruptedException e =
         assertThrows(SamInterruptedException.class, () -> samService.getSamUser());
     assertEquals("Request interrupted while getting user status info from Sam", e.getMessage());
     assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, e.getStatusCode());
   }
 
+  @Test
```

Comment:
These are the actual tests for this PR - ensuring that the proper user id is getting stored in the MDC and is included in the output message. This is true even when the user does not have permission to complete the action.